src/lib.rs

```diff
@@ -0,0 +1,36 @@
+#![crate_name = "librespot"]
```

Comment:
Aaaaah, thought it was the "recreate spotify" library :smile: Makes sense, thanks!